portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,16 @@
+  </head>
+  <body>
+    <div id="content">
+    <link rel="stylesheet" href="style.css">
+
+      <h1>Pictures of me</h1>
+      <div class="image-thumbnails">
+        <img src="/images/trasher.JPG"/>
+        <img src= "/images/rose.jpg">
+        <img src="/images/shaggy.JPG">
+        <img src="/images/prom1.jpg">
+        <img src="/images/IMG_1901.JPG">
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
it's also a good idea to keep a newline at the end of each file (the red marker indicates it isn't there)